inst/rmarkdown/templates/trb/resources/template.tex

```diff
@@ -0,0 +1,80 @@
+% Repository:  https://github.com/chiehrosswang/TRB_LaTeX_tex
+%
+% Transportation Research Board conference paper template
+% version 4.0 Lite (updates made to be compatible in Overleaf and ShareLaTeX)
+%
+%
+% When numbered option is activated, lines are numbered.
+\documentclass[numbered]{trbarticle}
+\usepackage{graphicx}
+\usepackage{booktabs}
+
+\newread\somefile
+\usepackage{xparse}
+
+\usepackage{natbib}
+\bibliographystyle{unsrtnat}
```

Comment:
You set `biblio-style` variable in YAML : this means it can be customized and should be this way: 
```suggestion
\bibliographystyle{$if(biblio-style)$$biblio-style$$else$unsrtnat$endif$}
```
 or with another defautl

Reply:
done